packages/cozy-harvest-lib/src/connections/accounts.js

```diff
@@ -0,0 +1,87 @@
+const ACCOUNTS_DOCTYPE = 'io.cozy.accounts'
+const PERMISSIONS_DOCTYPE = 'io.cozy.permissions'
+
+export const accountsMutations = client => {
+  const createAccount = async attributes => {
+    const { data } = await client.create(ACCOUNTS_DOCTYPE, attributes)
+    return data
+  }
+
+  const findParentAccount = async id => {
+    const { data } = await client.query(
+      client.find(ACCOUNTS_DOCTYPE).where({ _id: id })
+    )
+    return data[0]
+  }
+
+  const createParentAccount = id => {
```

Comment:
Again here, why is this specific to a parent account? It can't be juste `createAccountFromId`? They look like generic functions